docs/platform/release-notes/tizen-6-0-m2.md

```diff
@@ -0,0 +1,408 @@
+﻿# Tizen 6.0 Public M2
+
+Release date: Oct. 15, 2020
+
+
+## Release Notes
+
+### System (Kernel and System framework)
```

Comment:
@mijin85cho I had suggested capital F for the framework, although in the previous release notes, it is small F.

### System (Kernel and System Framework)


What is your opinion?

Reply:
@nawab-samsung Please keep the original

Reply:
sure